ui/components/NodeText.tsx

```diff
@@ -29,21 +31,27 @@ const Content: FC<Props> = ({ attributes }) => {
           data-testid={`node/text/${attributes.id}/text`}
         >
           <div className="row">
-            <List items={secrets} divided />
+            <List
+              items={secrets.map((item, k) => (
+                <span key={k}>{item}</span>
+              ))}
+              divided
+            />
             <div className="u-no-print">
               <Button
                 type="button"
-                onClick={() => {
-                  const text = secrets.join("\n");
-                  const element = document.createElement("a");
-                  const file = new Blob([text], {
-                    type: "text/plain",
-                  });
-                  element.href = URL.createObjectURL(file);
-                  element.download = "backup-codes.txt";
-                  document.body.appendChild(element);
-                  element.click();
-                  document.body.removeChild(element);
+                onClick={async () => {
```

Comment:
same suggestoin here to use `useCallback`

Reply:
Extracted and wrapped with use callback.